index.html

```diff
@@ -277,7 +303,7 @@ <h2 class="contact__title">
                     class="contact__form__textarea contact__form__inpt" 
                     maxlength="500" 
                     placeholder="Write your message here..." 
-                    rows="3" 
+                    rows="4"
```

Comment:
Great work on your contact section! I have some suggestions to help you get even closer to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122):

- [ ] It would be great if you could center your `"Get in Touch"` button to match the template. This **will create a visually pleasing alignment.** 🙏

- [ ] Additionally, please check the `width` of the button. Ensuring that the button has the correct `width` **will contribute to a more accurate representation of the desired design.** 🙋‍♂️

- [ ] Consider adding a bit of `margin-bottom` to the section. This will create some space between the contact section and the content below, **providing a better visual separation.** 😃

   **_These adjustments will help your contact section closely resemble the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122). Keep up the good work, and thank you for considering these suggestions!_**
   
   ### Your design:
   ![image](https://github.com/Feliperto/PortfolioWebsite-Microverse/assets/107952724/2c1ff75e-2be2-400c-a641-02acddaf73f9)

   ### Figma design:
   ![image](https://github.com/Feliperto/PortfolioWebsite-Microverse/assets/107952724/2fdcc0f4-6c06-4586-aa98-bb343a4440e3)